dillo/web/posts/routes.py

```diff
@@ -326,17 +341,31 @@ def render_page():
             'max_results': '15'
             }, api=api)
 
-        # Populate the feed
-        for post in latest_posts._items:
-            author = post.user.full_name
-            updated = post._updated if post._updated else post._created
-            url = url_for('posts.view', post_shortcode=post.properties.shortcode, community_url=post.project)
-            content = post.properties.content[:500]
-            feed.add(post.name, str(content),
-                     content_type='html',
-                     author=author,
-                     url=url,
-                     updated=updated,
-                     published=post._created)
+        populate_feed(feed, latest_posts)
         return feed.get_response()
     return render_page()
+
+
+@blueprint.route('/p/feeds/<community_url>/latest.atom')
+def feeds_blog(community_url):
```

Comment:
This is just being picky, but this would be the only place where we call community posts "blog". Maybe we can use feed_community, feed_all_communities ? Also feed instead of feeds, since it's only one feed we get.

Reply:
I was just following the original `feed_blogs` entry point, but to be honest I wasn't happy with the separation between them both either (as in, feeds_blogs vs feeds_blog). Let me change that

Reply:
and done